apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -83,6 +83,14 @@ export const BaseWorkflowByIdSchema = z.object({
       })
       .passthrough()
       .optional(),
+    pendingWorkflowEvents: z
```

Comment:
Used for ask for all re-uploads counter and sending an event for multiple workflows.